File: ABC/284/B.go
package main

import "fmt"

func main() {
	var t int
	fmt.Scan(&t)
	for i := 0; i < t; i++ {
		var n, cnt int
		fmt.Scan(&n)
		for j := 0; j < n; j++ {
			var a int
			fmt.Scan(&a)
			if a%2 == 1 {
				cnt++
			}
		}
		fmt.Println(cnt)
	}
}

File: ABC/284/b.rs
use proconio::input;
fn main() {
    input! {
        t: usize,
    }

    for _ in 0..t {
        input! {
            n: usize,
            a: [usize; n],
        }

        let mut cnt = 0;
        for v in a {
            if v % 2 == 1 {
                cnt += 1;
            }
        }
        println!("{}", cnt);
    }
}
